### ISSUE TITLE
bibliographic entry

### ISSUE THREAD

**Issue by millertg** [VandyVRC/tcadrt#12]
@wsalesky 
Bibliographic record is not displaying full source data. For example, Anchansi should show the full record for the Zhongguo wenwu dituji. Instead, it sites itself. Also, footnote display is confusing. See screenshot. 
![screen shot 2019-01-17 at 3 59 58 pm](https://user-images.githubusercontent.com/25234059/51351632-093b0400-1a71-11e9-867e-ac1e04336ae2.png)

**Comment by wsalesky** [comment]
@millertg how does it look now? If it looks okay, you can close the issue.

**Comment by millertg** [comment]
It is really close. I don't think the Bibliography should cite me, and certainly not Syriaca.org. Can we have the Bibliography show the Works Cited sources above?
![screen shot 2019-01-21 at 11 15 05 am](https://user-images.githubusercontent.com/25234059/51489439-ef9a0500-1d6d-11e9-8cd8-4dd96c18824e.png)

**Comment by millertg** [comment]
A second issue--I see the superscript number (1) in the Works Cited section, but I do not see the corresponding (1) in the text. In this case I think it should appear at the end of the last sentence (Ming dynasty. (superscript 1)).
![screen shot 2019-01-21 at 11 19 27 am](https://user-images.githubusercontent.com/25234059/51489611-7353f180-1d6e-11e9-9781-b16d26cb49ae.png)

**Comment by wsalesky** [comment]
@millertg 
1. The 'how to cite' section is for the article as a whole. I have removed you as the author and removed the reference to Syriaca.org. Does this work for you? I'm not sure you want the Works Cited entry here, as it is a citation for the TEI/HTML record that you have created. If you would still rather have the citation from the works cited section we can do that, I think you would need to add a source attribute to the listPlace element, so I can always get the correct tei:bibl record for the 'How to Cite' section. Review the changes and let me know how you want to proceed. 
 
2. This is a bug. I have fixed it: https://architecturasinica.org/place/000001
